lib/public/components/NavBar/index.js

```diff
@@ -81,7 +81,17 @@ const navBar = (model) => {
             pageTab(model, 'home', 'Home'),
             pageTab(model, 'log-overview', 'Log Entries'),
             pageTab(model, 'env-overview', 'Environments'),
-            pageTab(model, 'lhc-fill-overview', 'LHC Fills'),
+            h(`.navbar-dropdown${model.dropdownMenu === 'LHC' ? '.dropdown-open' : ''}`, [
```

Comment:
Several things here:
First, I think you can extract a `subMenu` or `dropdownSubMenu` component.
Then, you can probably use `dropdown` utitlity to clean this out.
Then, `pageLink` can probably use `frontLink`/`absoluteFrontLink`

Reply:
refactored

Reply:
What I meant by `Then, pageLink can probably use frontLink/absoluteFrontLink` is that:
You should have `pageLink` and `absoluteLink` and avoid to always have the construction of `?page=`
I would rather pass an `onClick` callback than the full model.

Reply:
refactored